k8s101.md

```diff
@@ -795,6 +795,9 @@ spec:
            name: mattermost-v1
 ```
 
+*NOTE:* Please remember to change the value of $registry with the actual value
```

Comment:
Nit: Give the command to extract the registry address like we do in prod patterns?

Reply:
Or try and turn into a oneliner?


Something along the lines of:
`cat mattermost/worker.yaml | sed 's/__registry__/$registry/g' | kubectl apply -f -`

Reply:
good idea. Added. Thanks :+1: